components/liveStream/liveStreamWidget.tsx

```diff
@@ -16,12 +16,14 @@ import { SpeakerInfo } from "../../services/server/events";
 import { CustomLink } from "../customLink";
 import { YouTubeEmbed } from "../embeds/youtubeEmbed";
 import { SocialIcons } from "../socialIcons/socialIcons";
-import { SubNewsLettersForm } from "../subNewsletter/subNewsletterForm";
+import { InlineJotForm } from "../blocks";
 
 type LiveStreamWidgetProps = {
   isLive?: boolean;
 } & LiveStreamProps;
 
+const NEWSLETTER_SUBSCRIBE_JOTFORM_ID = "240991361342051";
+
```

Comment:
This should be coming from Tina and maybe have a centralized place where we will define the ID for the newsletters component.

Reply:
Thanks Aman, i have moved this to Tina global index properties and referencing from there.